src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx

```diff
@@ -148,12 +168,33 @@ export const CoverHoverOverlay = styled.div`
 export const CoverIcon = styled(Icon)`
   transform: translateY(40px);
   transition: all ${transitions.timings.regular} ${transitions.easing};
-  width: 54px;
-  height: 54px;
   color: ${colors.white};
 `
-
-export const CoverPlayIcon = ({ ...props }) => <CoverIcon name="play-outline" {...props} />
+export const CoverPlayIcon = ({ ...props }) => (
```

Comment:
It seems the regular play icon was lost somewhere along the way :P 

<img width="402" alt="Screenshot 2021-02-25 at 13 13 54" src="https://user-images.githubusercontent.com/12646744/109152510-e4470e00-776b-11eb-9b60-dce3e7b8a33d.png">

Reply:
Uhh, was assuming `isAnyVideoSelected` was a `boolean` but in reality, it is `boolean | undefined`